cypress/pageObject/resources/ResourcesCreation.ts

```diff
@@ -0,0 +1,53 @@
+export class ResourcesCreation {
+  selectFacility(facility: string) {
+    cy.typeAndSelectOption('[data-cy="select-facility"]', facility, false);
+    return this;
+  }
+  selectStatus(status: string) {
+    cy.clickAndSelectOption('[data-cy="select-status-dropdown"]', status);
+    return this;
+  }
+  selectCategory(category: string) {
+    cy.clickAndSelectOption('[data-cy="select-category-dropdown"]', category);
+    return this;
+  }
+
+  selectAssignedUser(user: string) {
+    cy.typeAndSelectOption('[data-cy="select-assigned-user"]', user, false);
+    return this;
+  }
+  enterResourceTitle(title: string) {
+    cy.typeIntoField('[data-cy="title-input"]', title);
+    return this;
+  }
+
+  enterReasonOfRequest(reason: string) {
+    cy.typeIntoField('[data-cy="reason-input"]', reason);
+    return this;
+  }
+
+  clickFillMyDetails() {
+    cy.verifyAndClickElement(
+      '[data-cy="fill_my_details_button"]',
+      "Fill My Details",
+    );
+    return this;
+  }
+
+  verifyNameFilled(name: string) {
+    cy.get('[data-cy="contact_person"]').should("contain", name);
```

Comment:
```suggestion
    cy.get('[data-cy="contact_person"]').should("contain", name);
```

use verifycontentpresence command for all these kind of text verification